minigunapp/tasks.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from celery import shared_task
+from minigunapp.models import Email
+from django.core.mail import send_mail
+
+
+@shared_task
+def send_email(id):
+    """ Task that attempts to send the email with the given id.
+    Retries on a delay until max failures reached.
+    """
+
+    email = Email.objects.get(id=id)
+
+    try:
+        send_mail(
+            email.subject,
+            email.body,
```

Comment:
This is the django builtin send_mail, so to send messages in an html format, I'd use the html_message argument (https://docs.djangoproject.com/en/1.10/topics/email/#send-mail). Docs say this sends as a multipart/alternative with a text/plain part and a text/html part.